README.md

```diff
@@ -153,9 +153,12 @@ It creates a json file, `module-metadata-<timestamp>.json`, which contains an ar
 ```json
 [
   {
-    "name": "Module-name",
+    "moduleName": "Module-name",
     "total": "Total number of tests in the module",
-    "duration": "ms in Total duration to execute the module"
+    "passed": "A number of passed tests in the module",
+    "failed": "A number of failed tests in the module",
```

Comment:
hmm... i just realized, how about `skipped` tests?

Reply:
A number of skipped tests is not added to the module metadata.. do you think it's valuable information to be added? if so I can easily make the changes along in this pr

Reply:
Added a number of skipped tests to the metadata file :)